src/XamarinBackgroundKit.iOS/Renderers/MaterialContentViewRenderer.cs

```diff
@@ -55,7 +50,7 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 
         private void UpdateIsFocusable()
         {
-            if (ElementController.IsFocusable && ElementController.IsClickable)
+            if ((ElementController?.IsFocusable ?? false) && (ElementController?.IsClickable ?? false))
```

Comment:
Instead of checking ElementController for null twice, check if ElementController is null and return

Reply:
I have dynamic ContenViews, I remove and put them on the stack whenever I have new data, so I stay on the same page the error has occurred.